Branch-SDK/src/io/branch/referral/ShareLinkManager.java

```diff
@@ -45,6 +45,7 @@
     /* Indicates whether a sharing is in progress*/
     private boolean isShareInProgress_ = false;
 
+    @SuppressWarnings("deprecation")
```

Comment:
Why do we need to suppress a deprecation warning here and for all other mentions of ''ShareLinkBuilder"? This class doesn't appear to be deprecated?